product_classification_test_data/allposters/nodes.txt

```diff
@@ -0,0 +1 @@
":695,"display":"","strikethrough":"none","left":1217.25,"top":673,"right":1343,"visibility":""},"586":{"bottom":695,"display":"","strikethrough":"none","left":1217.25,"top":673,"right":1343,"visibility":""},"587":{"bottom":340,"display":"","strikethrough":"none","left":629.265625,"top":308,"right":694.28125,"visibility":""},"588":{"bottom":340,"display":"","strikethrough":"none","left":629.265625,"top":308,"right":694.28125,"visibility":""},"589":{"bottom":340,"display":"","strikethrough":"none","left":629.265625,"top":308,"right":694.28125,"visibility":""},"590":{"bottom":650,"display":"","strikethrough":"none","left":629.265625,"top":340,"right":745.921875,"visibility":""},"591":{"bottom":650,"display":"","strikethrough":"none","left":629.265625,"top":340,"right":745.921875,"visibility":""},"592":{"bottom":649,"display":"","strikethrough":"none","left":630.265625,"top":341,"right":744.921875,"visibility":""},"593":{"bottom":363,"display":"","strikethrough":"none","left":630.265625,"top":341,"right":744.921875,"visibility":""},"594":{"bottom":363,"display":"","strikethrough":"none","left":630.265625,"top":341,"right":744.921875,"visibility":""},"595":{"bottom":385,"display":"","strikethrough":"none","left":630.265625,"top":363,"right":744.921875,"visibility":""},"596":{"bottom":385,"display":"","strikethrough":"none","left":630.265625,"top":363,"right":744.921875,"visibility":""},"597":{"bottom":407,"display":"","strikethrough":"none","left":630.265625,"top":385,"right":744.921875,"visibility":""},"598":{"bottom":407,"display":"","strikethrough":"none","left":630.265625,"top":385,"right":744.921875,"visibility":""},"599":{"bottom":429,"display":"","strikethrough":"none","left":630.265625,"top":407,"right":744.921875,"visibility":""},"600":{"bottom":429,"display":"","strikethrough":"none","left":630.265625,"top":407,"right":744.921875,"visibility":""},"601":{"bottom":451,"display":"","strikethrough":"none","left":630.265625,"top":429,"right":744.921875,"visibility":""},"602":{"bottom":451,"display":"","strikethrough":"none","left":630.265625,"top":429,"right":744.921875,"visibility":""},"603":{"bottom":473,"display":"","strikethrough":"none","left":630.265625,"top":451,"right":744.921875,"visibility":""},"604":{"bottom":473,"display":"","strikethrough":"none","left":630.265625,"top":451,"right":744.921875,"visibility":""},"605":{"bottom":495,"display":"","strikethrough":"none","left":630.265625,"top":473,"right":744.921875,"visibility":""},"606":{"bottom":495,"display":"","strikethrough":"none","left":630.265625,"top":473,"right":744.921875,"visibility":""},"607":{"bottom":517,"display":"","strikethrough":"none","left":630.265625,"top":495,"right":744.921875,"visibility":""},"608":{"bottom":517,"display":"","strikethrough":"none","left":630.265625,"top":495,"right":744.921875,"visibility":""},"609":{"bottom":539,"display":"","strikethrough":"none","left":630.265625,"top":517,"right":744.921875,"visibility":""},"610":{"bottom":539,"display":"","s
```

Comment:
Decimals—interesting! I did not expect to see decimals in there. :-) I assume in a later commit you'll probably add defaults for a lot of these values (most things seem to be 0,0,0,0; not struck through; and visible), just to keep sizes and transmission costs down.